charts/rancher-istio/1.4.300/questions.yaml

```diff
@@ -1,3 +1,3 @@
 labels:
-  rancher.istio.v1.4.3: 1.4.3
-rancher_min_version: 2.3.4-rc1
+  rancher.istio.v1.4.300: 1.4.3
+rancher_min_version: 2.3.4
```

Comment:
let's set this for 2.3.4-rc1 so that anyone deploying a RC would get this chart version.

Reply:
fixed